content/en/apps/reference/app-settings/schedules.md

```diff
@@ -57,3 +57,25 @@ This sample shows a schedule with a single message, which will be sent on Monday
     }
 ]
 ```
+
+Following sample schedules a message for 270 days from `lmp_date`. If `lmp_date` doesn't exist on report, it will schedule a message for 270 days from `fields.lmp_date`. If both fields don't exist, it will create schedule from `reported_date`. This feature on `start_from` is supported from CHT 4.5.0+. 
```

Comment:
```suggestion
The following sample schedules a message for 270 days from `lmp_date`. If `lmp_date` doesn't exist on report, it will schedule a message for 270 days from `fields.lmp_date`. If neither field exists, it will create schedule from `reported_date`. Using an array for `start_from` is supported from CHT 4.5.0+. 
```